docs/en/observability/profiling-advanced-configuration.asciidoc

```diff
@@ -0,0 +1,11 @@
+[[profiling-advanced-configuration]]
+= Advanced configuration
+
+beta::[]
+
+After completing the steps in the <<profiling-get-started, Get started>> documentation, you may need to continue to more advanced configuration for your deployment.
+See the following sections for more information:
+
+* <<profiling-tag-data-query, Tag data for querying>>

```

Comment:
Nit: The order of this list and the menu on the left are different. Do we want to align them? IMHO a reasonable order is:

* Tag data for querying
* Add symbols for native frames
* Use a proxy